### DISCUSSION TITLE
How to recover my Google account in gmail

### DISCUSSION THREAD

**Discussion by malook1971** [community/community#129177] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Recover my Google account after reset my phone all Google account data deleted and I have forgotten password and I have tried many times and now Google shwoing massage many times tried services unavailable try after few hours and I have tried after more than 30 hours and still showing same massage.Please help me to recover my Google account.

**Comment by ossanna16** [comment]
Hi @malook1971,
Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Gmail. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
